ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml

```diff
@@ -3,25 +3,26 @@ base_images:
     name: ubi
     namespace: ocp
     tag: "8"
-  ocp_builder_golang-1.13:
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
     name: builder
     namespace: ocp
-    tag: golang-1.13
-  ocp_builder_golang-1.14:
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-1.16-openshift-4.8:
     name: builder
     namespace: ocp
-    tag: golang-1.14
+    tag: rhel-8-golang-1.16-openshift-4.8
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.14
+    tag: golang-1.16
 images:
 - dockerfile_path: Dockerfile.ci
+  from: base
```

Comment:
I don't think we can do this as the base gets overwritten from the upstream Dockerfile which results in the failures you see in AIDE can't be installed from UBI